### ISSUE TITLE
[Feature] Focus frame cast bar - "Show only interruptable spells"

### ISSUE THREAD

**Issue by PatriQ94** [Krealle/Cell_UnitFrames#124]
### Describe the feature

Focus is often used for tracking spells to interrupt. Would be great a way to distinguish in the castbar which spells can or cannot be interrupted so you don't waste an interrupt for nothing. 
Multiple possible ideas:
- Only show interruptable casts
- Highlight interruptable casts (so its easy to distinguish between the ones that can and those that can't be interrupted.

### Describe the implementation

Described above everything :)

**Comment by Krealle** [comment]
I like this idea. Will look into this once I get around to doing the already planned Cast Bar optimizations.

> Highlight interruptable casts (so its easy to distinguish between the ones that can and those that can't be interrupted.

This part however I'm a bit unsure what you mean, since you can already change the color of the Cast Bar based on if the cast is interruptible or not.

**Comment by PatriQ94** [comment]
> I like this idea. Will look into this once I get around to doing the already planned Cast Bar optimizations.
> 
> > Highlight interruptable casts (so its easy to distinguish between the ones that can and those that can't be interrupted.
> 
> This part however I'm a bit unsure what you mean, since you can already change the color of the Cast Bar based on if the cast is interruptible or not.

You're correct, found out that that part exists already after already posting.
